src/constants/classes.js

```diff
@@ -184,12 +185,25 @@ export const DROPDOWN_CLASSES = {
     MENU: 'chi-dropdown__menu',
     ITEM: 'chi-dropdown__menu-item',
     ITEM_TITLE: 'chi-dropdown__menu-item_title',
-    ITEM_DESCRIPTION: 'chi-dropdown__menu-item_text'
+    ITEM_DESCRIPTION: 'chi-dropdown__menu-item_text',
+    ICON: 'chi-icon icon-',
+    ICON_CHEVRON_RIGHT: 'chevron-right'
 };
 
 /* Skeleton */
 export const SKELETON_CLASSES = {
     SKELETON: 'chi-skeleton',
 };
 
+/* Steps */
+export const STEPS_CLASSES = {
+    SKELETON: 'chi-skeleton',
+    LINE: 'chi-steps__line',
+    ICON: 'chi-steps__icon',
+    CONTENT: 'chi-steps__content',
+    UX_PIN_WRAPPER: 'uxPin__wrapper',
```

Comment:
It is not a class exclusive to step, it is a common class that may be used in another component so it can be moved to the common section:  `export const UX_PIN_WRAPPER = 'uxPin__wrapper';`

```suggestion
```